File: src/main/java/com/matamercer/microblog/web/error/exceptions/AlreadyExistsException.java
package com.matamercer.microblog.web.error.exceptions;

public class AlreadyExistsException extends RuntimeException{
    public AlreadyExistsException() {
        super();
    }

    public AlreadyExistsException(final String message, final Throwable cause) {
        super(message, cause);
    }

    public AlreadyExistsException(final String message) {
        super(message);
    }

    public AlreadyExistsException(final Throwable cause) {
        super(cause);
    }
}

File: src/main/java/com/matamercer/microblog/web/error/exceptions/AlreadyExistsException.kt
package com.matamercer.microblog.web.error.exceptions

class AlreadyExistsException : RuntimeException {
    constructor() : super()
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(message: String?) : super(message)
    constructor(cause: Throwable?) : super(cause)
}